autoqct/chemicals.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+
+# Calculate the length of bond between two atoms
+def bondLength(atom1, atom2):
+    return np.sqrt(np.sum((atom1-atom2)**2))
```

Comment:
This is the easy way to work with all 3 components of a numpy vector at once (assuming atom1, atom2 are vectors).